dali_tf_plugin/setup.py.in

```diff
@@ -41,6 +41,7 @@ setup(name='nvidia-dali-tf-plugin@DALI_FLAVOR_MINUS@-cuda@CUDA_VERSION_SHORT_DIG
       author='NVIDIA Corporation',
       license='Apache License 2.0',
       packages=find_namespace_packages(include=['nvidia.*']),
+      package_data={str(find_namespace_packages(include=['nvidia.*'])[0]): ['Acknowledgements.txt', 'COPYRIGHT', 'LICENSE']},
```

Comment:
❌

Reply:
I think it is a must - https://docs.python.org/3.6/distutils/setupscript.html#installing-package-data. Otherwise only python files are included, and this case these are just a text one.